Stripe/Derived/Sources/TuistBundle+StripeiOS.swift

```diff
@@ -0,0 +1,24 @@
+// swiftlint:disable all
+// swift-format-ignore-file
+// swiftformat:disable all
+import Foundation
+
+// MARK: - Swift Bundle Accessor
+
+private class BundleFinder {}
+
+extension Foundation.Bundle {
```

Comment:
Do you know why this is needed? It looks like it's trying to recreate the SPM `Bundle.module` extension.

Reply:
It's autogenerated and I hadn't found a way to disable it, but I just did